vaadin-text-field-flow-parent/vaadin-text-field-flow/src/main/java/com/vaadin/flow/component/textfield/BigDecimalField.java

```diff
@@ -33,6 +33,7 @@
 import com.vaadin.flow.component.UI;
 import com.vaadin.flow.component.dependency.JsModule;
 import com.vaadin.flow.component.dependency.NpmPackage;
+import com.vaadin.flow.component.HasClearButton;
```

Comment:
nit: let's place this import before the `com.vaadin.flow.component.HasHelper` import.

Reply:
Done